driver-core/src/test/java/com/datastax/driver/core/ClusterInitTest.java

```diff
@@ -278,6 +278,69 @@ public void should_not_abort_init_if_host_does_not_support_protocol_version() {
       scassandraCluster.stop();
     }
   }
+  /**
```

Comment:
I didn't add a test for 1211 because is validated implicitly by now catching all runtime execptions in cluster init, and closing the cluster in various states of failed initialization in our other tests.